macosx/DefaultAppHelper.mm

```diff
@@ -0,0 +1,171 @@
+// This file Copyright © 2007-2023 Transmission authors and contributors.
+// It may be used under the MIT (SPDX: MIT) license.
+// License text can be found in the licenses/ folder.
+
+#import "DefaultAppHelper.h"
+
+#import <AppKit/AppKit.h>
+#import <UniformTypeIdentifiers/UniformTypeIdentifiers.h>
+
+static NSString* const kMagnetURLScheme = @"magnet";
+static NSString* const kTorrentFileType = @"org.bittorrent.torrent";
+
+UTType* GetTorrentFileType(void) API_AVAILABLE(macos(11.0))
```

Comment:
Any reason why it's a freestading func and not a class method?

Reply:
Just used example from the docs of [UTType.h](https://github.com/alexey-lysiuk/macos-sdk/blob/main/MacOSX14.0.sdk/System/Library/Frameworks/UniformTypeIdentifiers.framework/Versions/A/Headers/UTType.h#L373-L402)

Any reason to prefer class method?